ui/css/treeherder-details-panel.css

```diff
@@ -422,3 +422,32 @@ div.performance-panel {
   /* This is the same value as m-1 */
   margin: 0.25rem;
 }
+
+.performance-panel-data td, th {
+  white-space: nowrap;
+  /* The graph needs a little bit more padding compared to the defaults to be easily
+     readable.*/
+  padding-left: 0.6rem;
+  padding-right: 0.6rem;
+}
+
+.performance-panel-title {
+  font-size: 16px;
+  margin-top: 0.5rem;
+  line-height: 1.5;
+}
+
+/**
+ * This rule is a bit ugly, but the button doesn't fit in the table row without
+ * resizing the vertical contents. In addition, there isn't a button size that
+ * matches the font size of the row.
+ */
+.performance-panel-view-button {
+  padding: 0px 6px;
+  margin: -3px 0;
+  display: inline-block;
```

Comment:
Making this a `reactstrap` `Row` component might handle the horizontal display.

Reply:
I wrapped the `<a href>` in a row, and it doesn't really solve all the layout issues here compared to the custom CSS. I'd prefer to be precise in how this works around the bootstrap rules, as it's not clear to me that the rules in `Row` make it any simpler.

<img width="570" alt="image" src="https://user-images.githubusercontent.com/1588648/98955715-d6bf8e00-24c4-11eb-87cc-08aa786c57ba.png">

Reply:
Oh, I should have tried this.  Sorry.  I was way off on the `<Row>` element...

That being said, I tried this with the `<a>` element class having: `btn btn-outline-darker-secondary btn-sm performance-panel-view-button py-0` and then the `performance-panel-view-button` just setting the line-height to 17.  It looks like this for me:

![Screen Shot 2020-11-12 at 4 29 03 PM](https://user-images.githubusercontent.com/419924/99012901-3426ff80-2504-11eb-98b3-e14a10b832d6.png)

Reply:
> My process so far has been to design the space how I think it should look, and then I back off the custom style rules to hunt for bootstrap rules that make sense. Then in reviews, I'm getting lots of feedback on how to use even more bootstrap styles, but it degrades the initial designed layout. This doesn't really feel like an efficient process, and I'm not seeing how the bootstrap classes really help here.

It is certainly not my intention to make things harder for you to volunteer your time with us.  :). I apologize for the frustration there.  

The reasoning and motivation, such as they are, is that we ended up with a ton of complex and overlapping `CSS` in Treeherder that grew over the years and we're trying to reduce that now.  It ended up conflicting with itself at times and painted us into some corners.  So we're just trying to reuse what's already there in Bootstrap when possible and if it does the job right.  But you're already doing that here, and it totally makes sense that using custom CSS to make sure the job is done right is the way to go.  So thanks for bearing with my questions and suggestions.  :)

Reply:
Thanks for the explanation and context here. I re-read my comments, and they came across as a bit aggressive. Sorry about that. Thanks for your patience and help getting this merged!

Reply:
Thanks and no worries!  All is good.  :)